### PR TITLE
[11.0][MIG] sale_product_multi_add

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Standard migration of module sale_product_multi_add to v11 with slight modification.

Button has been moved from button box:
image

cc ~ @Eficent

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
ping @Cedric-Pigeon @rousseldenis

File: sale_product_multi_add/wizards/sale_import_products.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+from odoo import models, fields, api
+import odoo.addons.decimal_precision as dp
+
+
+class SaleImportProducts(models.TransientModel):
+    _name = 'sale.import.products'
+    _description = 'Sale Import Products'
+
+    products = fields.Many2many(comodel_name='product.product')
+    items = fields.One2many(comodel_name='sale.import.products.items',
```
Comment (Cedric-Pigeon) [inline sale_product_multi_add/wizards/sale_import_products.py:13]: `ondelete='cascade'` would be useful

**Review by Cedric-Pigeon — CHANGES_REQUESTED** [review]

**Review by Cedric-Pigeon — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by JordiBForgeFlow — APPROVED** [review]
:+1:  Technical and functional review.

**Pull request merged**